NEWS

```diff
@@ -2,7 +2,11 @@ Changes in 1.16.0
 ~~~~~~~~~~~~~~~~~~
 Released: not yet
 
-...
+Enhancements:
+
+ * Add the `FLATPAK_PTY_PROGRESS` environment variable, which allows enabling
+   an experimental feature where Flatpak sets the progress escape sequence. This
+   allows terminal emulators to show a progress indicator.
```

Comment:
This is a bit misleading, the feature isn't new (it was in 1.15.91). I'd say something more like

```
Bug fixes:

* Don't emit terminal progress indicator escape sequences by default.
    They are interpreted as notifications by some terminal emulators.
    (#6052)

Enhancements:

* Add the `FLATPAK_TTY_PROGRESS` environment variable, which re-enables
     the terminal progress indicator escape sequences added in 1.15.91.
```

Reply:
I removed the NEWS change from this PR. Will update after it's merged.